### ISSUE TITLE
Cannot find .NET Core SDK 1.0.3 on Docker

### ISSUE THREAD

**Issue by tugberkugurlu** [dotnet/dotnet-docker#232]
I have installed .NET tooling for Mac from https://www.microsoft.com/net/core#macos and I have ended up with v1.0.3 SDK:

```
➜  apps dotnet --info
.NET Command Line Tools (1.0.3)

Product Information:
 Version:            1.0.3
 Commit SHA-1 hash:  37224c9917

Runtime Environment:
 OS Name:     Mac OS X
 OS Version:  10.12
 OS Platform: Darwin
 RID:         osx.10.12-x64
 Base Path:   /usr/local/share/dotnet/sdk/1.0.3
```

I can only see SDK v1.0.3 installed, nothing else:

```
➜  apps ls /usr/local/share/dotnet/sdk
1.0.3
```

When I try to use the docker image `microsoft/dotnet:sdk` or `microsoft/dotnet:1.0.4-sdk` or `FROM microsoft/dotnet:1.1.1-sdk`, I don't happen to get SDK v1.0.3. 

```
Step 14/21 : RUN dotnet --info
 ---> Running in 0730746a15ce
.NET Command Line Tools (1.0.1)

Product Information:
 Version:            1.0.1
 Commit SHA-1 hash:  005db40cd1

Runtime Environment:
 OS Name:     debian
 OS Version:  8
 OS Platform: Linux
 RID:         debian.8-x64
 Base Path:   /usr/share/dotnet/sdk/1.0.1
 ---> e0db57dd76cc
Removing intermediate container 0730746a15ce
Step 15/21 : RUN ls /usr/share/dotnet/sdk
 ---> Running in 569c3d4ad62c
1.0.1
 ---> 2ebf612a96a8
```

and there doesn't seem to be a tag for v1.0.3 (only `1.0.3-sdk-msbuild-rc4` which I imagine is not what I want).

Is v1.0.3 is the latest stable? If so, how can I get it on docker? Is there a place that I can see the versions of available SDKs? Do I have a chance to install other versions of SDKs side by side and if so, how?

**Comment by MichaelSimons** [comment]
The latest LTS release is 1.0.4.  You can see the released versions [here](https://github.com/dotnet/core/tree/master/release-notes/download-archives).  You can install multiple SDKs side by side.

**Comment by tugberkugurlu** [comment]
Thanks! So, is it the case that https://www.microsoft.com/net/download/core is not updated? I am getting 1.0.3 SDK there, not 1.0.4.

**Comment by MichaelSimons** [comment]
This issue should have been resolved with the 1.0.5 release.  Please upgrade and let us know if this is still causing issues.
